### PR TITLE
Fixed #2531 -- added missing EV oid

### LINKED ISSUE #2531: NameOID should have EV OIDs
We're at least missing `jurisdictionCountryName` (1.3.6.1.4.1.311.60.2.1.3). This OID was formerly called `jurisdictionOfIncorporationCountryName` (changed in this CAB ballot https://cabforum.org/2014/03/24/ballot-119-remove-ofincorporation-from-oid-descriptions-in-evg-9-2-5/)

### PR DESCRIPTION
**Pull request by alex**
If there are others missing let me know, I'll add them to this PR, otherwise we can just do them as they come up.

### CODE REVIEW

**Comment by reaperhulk** [comment]
We need businessCategory, incorporating locality (1.3.6.1.4.1.311.60.2.1.1), incorporating state/province (1.3.6.1.4.1.311.60.2.1.2), and incorporating country (1.3.6.1.4.1.311.60.2.1.3).

**Comment by alex** [comment]
Do you have the OID for businessCategory?

On Sat, Dec 19, 2015 at 1:33 PM, Paul Kehrer <notifications@github.com>
wrote:

> We need businessCategory, incorporating locality
> (1.3.6.1.4.1.311.60.2.1.1), incorporating state/province
> (1.3.6.1.4.1.311.60.2.1.2), and incorporating country
> (1.3.6.1.4.1.311.60.2.1.3).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2534#issuecomment-166012911>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084

**Comment by reaperhulk** [comment]
2.5.4.15 is the businessCategory OID

**Comment by alex** [comment]
@reaperhulk added

**Comment by reaperhulk** [comment]
These should have short names added to the `_OID_NAMES` as well.

**Comment by alex** [comment]
@reaperhulk good call, adding

**Comment by alex** [comment]
done.

On Sat, Dec 19, 2015 at 8:54 PM, Paul Kehrer <notifications@github.com>
wrote:

> These should have short names added to the _OID_NAMES as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2534#issuecomment-166053621>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084

File: src/cryptography/x509/oid.py [hunk 1]
```diff
@@ -151,6 +157,17 @@ class CertificatePoliciesOID(object):
     NameOID.PSEUDONYM: "pseudonym",
     NameOID.DOMAIN_COMPONENT: "domainComponent",
     NameOID.EMAIL_ADDRESS: "emailAddress",
+    NameOID.JURISDICTION_COUNTRY_NAME: (
+        "jurisdictionOfIncorporationCountryName"
```
Comment (reaperhulk) [inline src/cryptography/x509/oid.py]: Per https://cabforum.org/2014/03/24/ballot-119-remove-ofincorporation-from-oid-descriptions-in-evg-9-2-5/ these OIDs no longer have `OfIncorporation` in their short names. (Kind of a pedantic point given the difficulty in having canonical names for OIDs I guess)

**Comment by reaperhulk** [comment]
LGTM, will merge once tests pass.

**Pull request merged**